models/sessions/scratch/bigquery/snowplow_web_sessions_this_run.sql

```diff
@@ -192,6 +226,17 @@ select
     -- engagement fields
     c.page_views,
     c.engaged_time_in_s,
+    {%- if var('snowplow__list_event_counts', false) %}
+    SAFE.PARSE_JSON(c.event_counts_string) as event_counts,
+    {% endif %}
+    c.total_events,
+    c.page_views >= 2 or c.engaged_time_in_s / {{ var('snowplow__heartbeat', 5) }} >= 2
```

Comment:
Imposing our engagement logic onto the user here

Reply:
Yeah we've discussed maybe breaking this out into a macro, but for now are going to leave it as a defined thing and then can split it out later if we get feedback someone wants to customise it